### DISCUSSION TITLE
Make the design system available for Vue JS

### DISCUSSION THREAD

**Discussion by philippbeck** [coingaming/moon-design#2503] [Ideas]
Hey guys, is it possible to make the Design System available for Vue JS?

**Comment by ianhawes** [comment]
I'd also be interested in a port of this design system to Vue 3.

**Comment by karl-kallavus** [comment]
Hey guys,

While we currently don't have the resources to provide a dedicated Design System for Vue JS or a port to Vue 3, we appreciate your interest. 

If you have any specific questions or if you're looking for sponsorship opportunities to support this development, please feel free to keep the discussion within this thread. We'll do our best to address your inquiries.

Thank you for understanding!
